src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -476,6 +476,14 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		}
 	}
 
+	public sendNotebookTelemetryEvent(action: TelemetryKeys.TelemetryAction | TelemetryKeys.NbTelemetryAction, additionalProperties: ITelemetryEventProperties = {}): void {
+		let properties: ITelemetryEventProperties = additionalProperties;
```

Comment:
We should be deep cloning this so we aren't modifying what the user passed in.

(and FYI creating a new var like this is unnecessary since it's still pointing at the same object as additionalProperties)